### PR TITLE
DEV-8320: Issues fixes

### PR DESCRIPTION
**Pull request by sdorleTVPage**

### CODE REVIEW

File: layouts/partials/carousel/host-mobile.css [hunk 1]
```diff
@@ -66,6 +66,10 @@
     "overflow-y: auto;" +
     "opacity: 1;" +
     "visibility: visible;" +
+    "align-items: center;" +
+    "align-content: center;" +
+    "justify-content: center;" +
+    "display: {{ ($.Scratch.Get "settings").video_modal_center_display }};" +
```
Comment (rfornes) [inline layouts/partials/carousel/host-mobile.css]: `.tvp-modal-wrapper.centered {
    position: absolute;
    top: 50%;
    left: 0;
    right: 0;
    transform: translateY(-50%);
    margin: auto;
}`

File: static/libs/player.js [hunk 2]
```diff
@@ -146,7 +146,7 @@
     var that = this;
     if (window.location !== window.parent.location && iOS) {
       var onHolderResize = function(e) {
-        if(!e || !e.data || !e.data.event || that.eventPrefix + ':holder_resize' !== e.data.event)
+        if(!e || !e.data || !e.data.event || that.eventPrefix + ':modal_holder_resize' !== e.data.event)
```
Comment (rfornes) [inline static/libs/player.js]: Lets rename to read: `:external_resize`

File: static/sidebar/js/grid.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
     return (window.innerWidth || document.documentElement.clientWidth || body.clientWidth) <= 200 ? 'small' : 'medium';
   };
 
-  var Grid = function (el, options) {
```
Comment (rfornes) [inline static/sidebar/js/grid.js]: Revert

File: static/inline/js/inline.js [hunk 4]
```diff
@@ -222,7 +222,6 @@
                         settings: {
                             arrows: false,
                             centerPadding: generalOptions.product_holder_slide_center_padding,
-                            centerMode : true,
```
Comment (rfornes) [inline static/inline/js/inline.js]: When there's only 1 product no need to run the slider as it brakes the layout

File: layouts/partials/inline-spotlight/inline.css [hunk 5]
```diff
@@ -419,4 +419,25 @@
 	'.tvp-product-info {'+
     'display: {{- ($.Scratch.Get "settings").product_item_info_display_mobile -}};' +
 	'}'+
+	'.tvp-slide-up {'+
+    'display: {{- ($.Scratch.Get "settings").product_item_show_hide_feature_mobile_display -}};' +
+	'}'+
+	'.tvp-slide-up > .tvp-minus {'+
```
Comment (rfornes) [inline layouts/partials/inline-spotlight/inline.css]: Please merge the rules to have one css declaration

File: static/libs/player.js (hunk 2 above)
Comment (sdorleTVPage) [reply static/libs/player.js]: done

File: layouts/partials/inline-spotlight/inline.css (hunk 5 above)
Comment (sdorleTVPage) [reply layouts/partials/inline-spotlight/inline.css]: done

File: static/inline/js/inline.js (hunk 4 above)
Comment (sdorleTVPage) [reply static/inline/js/inline.js]: done

**Pull request closed without merging**